cvat/apps/consensus/redis_migrations/001_migrate_to_key_value_id_format.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import re
+
+from django.conf import settings
+
+from cvat.apps.redis_handler.redis_migrations import AbstractJobProcessor, BaseMigration
+from cvat.apps.redis_handler.redis_migrations.utils import get_job_func_name
+
+
+class JobProcessor(AbstractJobProcessor):
+    def __call__(self, job, *, logger, registry):
+        if get_job_func_name(job) != "_merge":
+            raise self.UnexpectedJobError()
+
+        # previous ID formats:
+        # consensus-merge-task-<task_id>-user-<user_id>
+        # consensus-merge-task-<task_id>-job-<job_id>-user-<user_id>
+        for pattern in (
+            r"consensus-merge-(?P<target>task)-(?P<target_id>\d+)-user-\d+",
+            r"consensus-merge-task-\d+-(?P<target>job)-(?P<target_id>\d+))-user-\d+",
+        ):
+            if match := re.fullmatch(pattern, job.id):
+                matched = match.groupdict()
+                job.id = f"action=merge&target={matched['target']}&target_id={matched['target_id']}"
```

Comment:
I don't quite understand how these ID changes are meant to work. Presumably `job.save` will save the job under the new key, but then:

1. Wouldn't all the queues and registries still refer to the old job ID?
2. Won't the old key remain in Redis?

Reply:
Looks like I screwed up :)
I thougth about handling job dependencies/dependent jobs, but forgot about this..